docs/API.md

```diff
@@ -195,21 +203,28 @@ The object has three keys, `"module"`, `"message"` and `"level"`.
 
 Sometimes additional keys are present.
 
-* `"ns"`: a *domain name*. The name server used by the *test module*.
-
->
-> TODO: Can other extra keys in addition to `"ns"` occur here? Can something be said
-> about when each extra key is present?
->
+* `"ns"`: a *domain name*. The name server used by the *test module*. 
+This key is added when the module name is `"NAMESERVER"`.
 
 
 ### Timestamp
 
 Basic data type: string
 
->
-> TODO: Specify date format
->
+Default database timestamp format. 
```

Comment:
The timestamp format should be database engine independent. We could even require that all timestamps be in UTC.

Reply:
As the Timestamp data type is only used in API responses and doesn't require any understanding by anyone else that the API users I would leave this as it was:  "Default database timestamp format" with a possible rewording as so: "A time in a format defined by your local database setup and configuration". Otherwise the backend needs updates to force this to be something standard.

Reply:
I still think that the timestamp format should be specified.